utils/src/main/java/org/apache/cloudstack/utils/bytescale/ByteScaleUtils.java

```diff
@@ -27,32 +27,32 @@ public class ByteScaleUtils {
     private ByteScaleUtils() {}
 
     /**
-     * Converts mebibytes to bytes.
+     * Converts mebibyte to bytes.
      *
      * @param mib The value to convert to bytes (eq: 1, 2, 3, ..., 42,...).
-     * @return The parameter multiplied by 1048576 (1024 * 1024, 1 MiB).
+     * @return The parameter multiplied by 2²⁰ (1048576 | 1024 * 1024 | 1 MiB | ).
      */
     public static long mibToBytes(long mib) {
```

Comment:
@weizhouapache, what do you think about not using the symbols (e.g.: `mebibytesToBytes`) ? This way it would leave no doubt and would be consistent, as it is written `...ToBytes`, and not `...toB`.

Reply:
@GutoVeronezi looks good to me.
@DaanHoogland what is your opinion?

Reply:
I like @GutoVeronezi 's last proposal here, reading will actually be quicker as you don´t need to translate in your head.